numba/tests/usecases.py

```diff
@@ -2,6 +2,8 @@
 import numpy as np
 from numba import jit
 
+_GLOBAL_STR = "abc"
```

Comment:
As a total aside, this file itself seems a little strange with respect to the rest of testing, is it perhaps a legacy item?

Reply:
This file along with `test_usecases.py` are probably the oldest surviving test files and they are exercising the essentials of the core compiler.